src/main/java/educonnect/logic/commands/HelpCommand.java

```diff
@@ -16,6 +16,6 @@ public class HelpCommand extends Command {
 
     @Override
     public CommandResult execute(Model model) {
-        return new CommandResult(SHOWING_HELP_MESSAGE, true, false);
+        return new CommandResult(SHOWING_HELP_MESSAGE, true, false, false);
```

Comment:
I think SHOWING_HELP_MESSAGE can be updated to show that timetable is a parameter because it is just a small update.